apps/studio/components/layouts/ProjectIntegrationsLayout/ProjectIntegrationsLayout.tsx

```diff
@@ -31,7 +31,7 @@ const ProjectIntegrationsMenu = () => {
   const pgNetExtensionExists = (data ?? []).find((ext) => ext.name === 'pg_net') !== undefined
   const graphqlExtensionExists = (data ?? []).find((ext) => ext.name === 'pg_graphql') !== undefined
   // TODO: Change this to true for local development to work
-  const pgmqExtensionExists = (data ?? []).find((ext) => ext.name === 'pgmq') !== undefined
+  const pgmqExtensionExists = true // (data ?? []).find((ext) => ext.name === 'pgmq') !== undefined
```

Comment:
Does this one need to be commented out for testing only as well?
```suggestion
  const pgmqExtensionExists = (data ?? []).find((ext) => ext.name === 'pgmq') !== undefined
```

Reply:
yup! will need to revert at the end